README.md

```diff
@@ -2,78 +2,65 @@
-  
-  10.  **`package-lock.json`** (See `package.json` below, first). This is an automatically generated file based on the exact versions of your npm dependencies that were installed for your project. **(You won’t change this file directly).**
-  
-  11.  **`package.json`**: A manifest file for Node.js projects, which includes things like metadata (the project’s name, author, etc). This manifest is how npm knows which packages to install for your project.
-  
-  12.  **`README.md`**: A text file containing useful reference information about your project.
-  
-  13.  **`yarn.lock`**: [Yarn](https://yarnpkg.com/) is a package manager alternative to npm. You can use either yarn or npm, though all of the Gatsby docs reference npm.  This file serves essentially the same purpose as `package-lock.json`, just for a different package management system.
-
-## 🎓 Learning Gatsby
-
-Looking for more guidance? Full documentation for Gatsby lives [on the website](https://www.gatsbyjs.org/). Here are some places to start:
-
--   **For most developers, we recommend starting with our [in-depth tutorial for creating a site with Gatsby](https://www.gatsbyjs.org/tutorial/).** It starts with zero assumptions about your level of ability and walks through every step of the process.
-
--   **To dive straight into code samples, head [to our documentation](https://www.gatsbyjs.org/docs/).** In particular, check out the _Guides_, _API Reference_, and _Advanced Tutorials_ sections in the sidebar.
-
-## 💫 Deploy
-
-[![Deploy to Netlify](https://www.netlify.com/img/deploy/button.svg)](https://app.netlify.com/start/deploy?repository=https://github.com/gatsbyjs/gatsby-starter-default)
+
+    Save your changes and the browser will update in real time!
+
+# 👷‍♀️ Maintenance
+
+Please use `yarn` to manage the dependencies and scripts in this project. You shouldn't have to worry about which version you have, so long as it is better than 1.0.1 this system will take care of the details.
+
+1. **Keeping your local copy up to date**
+
+    It's a good idea check that your node version and dependencies are in sync. When you pull from a branch we suggest running these commands to be sure:
+
+    ```sh
+    nvm use
+    yarn
+    ```
+
+1. **Adding dependencies**
+
+    The best way to add a new dependency is via `yarn` as follows:
+
+    ```sh
+    yarn add --exact some-cool-package
+    ```
+
+    If the package is part of the test or build tools for this project please be sure to add `--dev`.
+
+    **why `--exact`? To avoid problems with dependencies updating unexpectedly because of semver masking we want to "pin" them to a specific version number. This'll also help our robots when they want to tell us about updates.*
+
+1. **Maintaining dependencies**
+
+    If you want to check for, and update to, new versions of our dependencies manually the easiest way to do so is by running
+
+    ```sh
+    yarn upgrade-interactive --latest
```

Comment:
I recommend we do this frequently. On sprint based projects we try to upgrade at the start of each sprint. Is it worth noting/carding that somewhere?

Reply:
Ah - of course. :woman_facepalming: